samples/nrf5340/smp_svr/README.rst

```diff
@@ -0,0 +1,185 @@
+    nrfjprog -f NRF53 --sectorerase --program mcuboot/zephyr/zephyr.hex --verify
+    nrfjprog -f NRF53 --sectorerase --program zephyr/internal_flash_signed.hex --verify
+    nrfjprog -f NRF53 --qspisectorerase --program zephyr/qspi_flash_signed.hex --qspiini <path_to_sample>/Qspi_thingy53.ini --verify
+    nrfjprog -f NRF53 --reset
+
+Sample image: Hello world!
+--------------------------
+
+The ``smp_svr`` app is now ready to run.
+Reset your board and test the app with the :file:`mcumgr` command-line tool's ``echo`` functionality.
+ This will send a string to the remote target device and have it echo it back:
+
+.. tabs::
+
+   .. group-tab:: Bluetooth®
+
+      .. code-block:: console
+
+         sudo mcumgr --conntype ble --connstring ctlr_name=hci0,peer_name='Zephyr' echo hello
+         hello
+
+   .. group-tab:: Shell
+
+      .. code-block:: console
+
+         mcumgr --conntype serial --connstring "/dev/ttyACM2,baud=115200" echo hello
+         hello
+
+.. note::
+   The :file:`mcumgr` command-line tool requires a connection string in order to identify the remote target device.
+   In the Bluetooth version of the sample a Bluetooth LE-based connection string is used, and you may need to modify it depending on the Bluetooth LE controller you are using.
+
+   In the following sections, examples will use ``<connection string>`` to represent
+   the ``--conntype <type>`` and ``--connstring=<string>`` :file:`mcumgr` parameters.
+
+
+Image upload and image mapping to MCUboot slot
+----------------------------------------------
+
+For this sample configuration the mcumgr supports, for upload, 3 target images, of which allows to update all parts of the firmware.
+
+The mcumgr ``image upload`` command may be provided optional ``-e -n <image>`` parameter that will
+select target image for upload; when parameter is no provided, 0 is assumed, which means "default
+behaviour", and it performs upload to the "image-1", the MCUboot secondary slot.
+
+The Partition manager (PM) label for slot-to-``<image>`` translation is shown in the table below:
+
+    +-----------------------+--------+-------------+---------------------------+
+    | PM label              | Slot   | -n <image>  |       Firmware part       |
+    +=======================+========+=============+===========================+
+    | "mcuboot_secondary"   | slot-1 |     0       | internal application part |
+    +-----------------------+--------+-------------+---------------------------+
+    | "mcuboot_secondary_1" | slot-3 |     1       | networking                |
+    +-----------------------+--------+-------------+---------------------------+
+    | "mcuboot_secondary_2" | slot-5 |     2       | qspi application part     |
+    +-----------------------+--------+-------------+---------------------------+
+
+.. note::
+
+   The ``-e`` option means "no erase", and is provided to the Mcumgr to prevent it from sending the erase command to target before updating image.
```

Comment:
```suggestion
   The ``-e`` option means "no erase", and is provided to the MCUmgr to prevent it from sending the erase command to the target before updating the image.
```